sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/ManagementGroup/Models/ManagementGroupInfoData.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.Core
+{
+    /// <summary> The management group resource. </summary>
+    public partial class ManagementGroupInfoData
```

Comment:
Decided to go with MGInfoData for this, rather than MGExpanded for the class with additional properties?

Reply:
I wanted the good name to be the main case which is ManagementGroup.  I have this as an open question during archboard to agree on naming here